attractors_pick: [-0.3,  0.3, 0.25, -0.2,  0.5, 0.25, -0.4,  0.5, 0.25]
attractor_sink:  [ 0.3, 0.3,  0.25]
